businesstime/australia.py

```diff
@@ -0,0 +1,101 @@
+
+from datetime import date
+from businesstime import Holidays
+
+class QueenslandPublicHolidays(Holidays):
+    """
+    http://www.qld.gov.au/recreation/travel/holidays/public/
+
+    Current coverage is only 2013-2016.
+    """
+
+    # Australian public holidays are legislated by each state individually for 
```

Comment:
Users can already define their own holidays, either as a list or as a generator. This is basically just a convenience addition and has the nice benefit of reminding you with a `NotImplementedError` when the list needs an update.